apps.yaml

```diff
@@ -168,6 +168,7 @@ appsInfo:
     integration: Otomi has security best practices built in, and is designed for intrusion. Istio is used by Otomi as a service mesh to deliver mTLS enforcement for all traffic that is deemed compromisable, egress control to force teams to choose explicit egress endpoints, and advanced routing capabilities such as weight based load balancing (A/B or blue/green testing). Istio is part of the core of Otomi and can not be disabled.
   jaeger:
```

Comment:
Why do we still need Jaeger if traces can be visible in Grafana ?

Reply:
No. Only when Tempo is enabled, grafana datasources are configured. For now you can use both (Jaeger and Tempo/Grafana), but we could (later) make the user choose and if Jaeger is used, configure Grafana to add a link to a traceid in the logs (Loki) and open the trace in Jaeger